docs/src/index.md

```diff
@@ -1,3 +1,15 @@
 # ManifoldDiff
 
 The package __ManifoldDiff__ aims to provide automatic calculation of Riemannian gradients of functions defined on manifolds. It builds upon [`Manifolds.jl`](https://github.com/JuliaManifolds/Manifolds.jl).
+
+## Naming scheme
+
+Providing a derivative, differential or gradient for a given function, this package adds that information to the function name.
+For example
+
+* `grad_f` for a gradient ``\operatorname{grad} f``
+* `differential_f` for ``Df``
+* `differential_f_variable` if `f` has mutliple variables / parameters, since a usual writing in math is ``f_x`` in this case
```

Comment:
```suggestion
* `differential_f` for ``Df`` (also called pushforward)
* `differential_f_variable` if `f` has multiple variables / parameters, since a usual writing in math is ``f_x`` in this case
* `adjoint_differential_f` for pullbacks
* `adjoint_differential_f_variable` if `f` has multiple variables / parameters
```
Are my additions are correct?